### ISSUE TITLE
Unable to move history page

### ISSUE THREAD

**Issue by nyoronjp** [support-project/knowledge#938]
1. Select an item in list.
   - https://service-knowledge.support-project.org/open.knowledge/histories/19
2. Display error message.

```json
{
    children: [ ],
    code: 400,
    message: "param is invalid.",
    result: null,
    status: 2
}
```

Beccause URL is invalid.
`/open.knowledge/history/19&page=0&history_no=2`
Correctly
`/open.knowledge/history/19?page=0&history_no=2`

**Comment by koda-masaru** [comment]
Thank you for your advice.
I fixed this bug on #940

I am developing the next version. (v1.12.0 dev)
And, I will deploy the version under development to the test environment.
Would you advise me If you find a bug.

**Comment by nyoronjp** [comment]
LGTM 👍
